### ISSUE TITLE
Please add release tags

### ISSUE THREAD

**Issue by tillea** [rambaut/jam-lib#2]
Hi,
jam-lib is packaged for Debian.  It would be helpful if you could ass release tags which would enable us to automatically detect updates of the code to always provide the latest version to our users.
Thanks for considering, Andreas.

**Comment by emollier** [comment]
Hi,

I  agree, this is currently worked around in Debian by deriving a pseudo version number from the latest git commit.  The package libjam-java currently has version 0.1.git20180106.740247a+dfsg-1, and the upcoming upload will have version 0.1.git20240425.aec45be+dfsg-1.  Technically, it works kind of okay, but it is hard to parse for human beings.

Have a nice day,  :)
Étienne.

**Comment by tillea** [comment]
Am Fri, Oct 11, 2024 at 02:30:19AM -0700 schrieb Étienne Mollier:
> I  agree, this is currently worked around in Debian by deriving a pseudo version number from the latest git commit.  The package libjam-java currently has version 0.1.git20180106.740247a+dfsg-1, and the upcoming upload will have version 0.1.git20240425.aec45be+dfsg-1.  Technically, it works kind of okay, but it is hard to parse for human beings.

Besides being hard to read we never know whether the latest commit is intended to be used by a wider audience or a random development commit.  It would be nice to know what is of production quality.
